CadRevealFbxProvider.Tests/BatchUtils/ScaffoldPartOptimizers/ScaffoldPartOptimizerTestPartA.cs

```diff
@@ -15,9 +15,9 @@ public override List<uint> GetIndicesTruth()
         return [1, 0, 2];
     }
 
-    public override string GetName()
+    public override string Name
     {
-        return "Part A test optimizer";
+        get { return "Part A test optimizer"; }
     }
```

Comment:
You could do something like this:
`public override string Name {get;} = "Part A test optimizer";`